apps/ledger-live-mobile/src/screens/BleDevicePairingFlow/index.tsx

```diff
@@ -70,7 +74,9 @@ export const BleDevicePairingFlow: React.FC<Props> = ({ navigation }) => {
       <Flex px={6} flex={1}>
         <BleDevicePairingFlowComponent
           key={keyToReset}
-          filterByDeviceModelId={DevicesWithTouchScreen}
+          filterByDeviceModelId={DevicesWithTouchScreen.filter(
+            deviceModelId => isApexSupported || deviceModelId !== DeviceModelId.apex,
+          )}
```

Comment:
This is not 100% necessary as there's no way to scan an Apex device if you don't already have one. The FF is there to prevent users to go into paths that are not yet usable.